### DISCUSSION TITLE
Welcome to Nicegram Discussions!

### DISCUSSION THREAD

**Discussion by sshlg** [nicegram/Telegram-iOS#165] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by MrSmart0401** [comment]
Hello Sergey S!
This is Jorge, nice to contact with you.

**Reply by ukoemchhai-cmd** [reply]
T
